<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AidenTrader v0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pytrader.png</normaloff>pytrader.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>326</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>356</width>
      <height>426</height>
     </size>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QPushButton" name="orderBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>189</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>65</y>
       <width>24</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
     <property name="buddy">
      <cstring>codeLineEdit</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>43</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>44</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
     <property name="buddy">
      <cstring>accountComboBox</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="orderTypeComboBox">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
     <property name="buddy">
      <cstring>orderTypeComboBox</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>115</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
     <property name="buddy">
      <cstring>hogaTypeComboBox</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="hogaTypeComboBox">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>115</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>139</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
     <property name="buddy">
      <cstring>qtySpinBox</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="qtySpinBox">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>139</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>163</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
     <property name="buddy">
      <cstring>priceSpinBox</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="priceSpinBox">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>163</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>66</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeNameLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>150</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255)</string>
     </property>
    </widget>
    <zorder>accountComboBox</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>hogaTypeComboBox</zorder>
    <zorder>label_5</zorder>
    <zorder>qtySpinBox</zorder>
    <zorder>label_6</zorder>
    <zorder>priceSpinBox</zorder>
    <zorder>codeLineEdit</zorder>
    <zorder>codeNameLineEdit</zorder>
    <zorder>orderTypeComboBox</zorder>
    <zorder>orderBtn</zorder>
    <zorder>label_3</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>실시간 Trading</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>24</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
     <property name="buddy">
      <cstring>codeLineEdit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSilsiganJongmok">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSilsiganJongmok">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>22</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeNameLineEditSilsiganJongmok">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>45</y>
       <width>150</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>323</y>
      <width>661</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>실시간매매</string>
    </property>
    <widget class="QTableWidget" name="realTimeTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>641</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>전일대비</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>체결량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>시가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>고가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>저가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매도호가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매수호가</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>624</y>
      <width>1021</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>로그</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QTextEdit" name="logTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>661</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1289</width>
      <height>876</height>
     </size>
    </property>
    <property name="title">
     <string>잔고 및 보유종목 현황</string>
    </property>
    <widget class="QTableWidget" name="accountEvaluationTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>641</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1251</width>
       <height>111</height>
      </size>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금 (d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률 (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="stocksTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>641</width>
       <height>201</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1251</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유량</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매입가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>평가손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수익률 (%)</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="realtimeCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>290</y>
       <width>87</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inquiryBtn">
     <property name="geometry">
      <rect>
       <x>577</x>
       <y>284</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>488</y>
      <width>661</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>자동 선정 종목 리스트</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableWidget" name="automatedStocksTable">
       <column>
        <property name="text">
         <string>주문유형</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>호가구분</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>가격</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>상태</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>자동매매종목</string>
     </attribute>
     <widget class="QToolButton" name="toolButtonAddJongmok">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>add.png</normaloff>add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="popupMode">
       <enum>QToolButton::DelayedPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButtonRemoveJongkmok">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>40</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Minus.png</normaloff>Minus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetAutoJongmok">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>accountComboBox</tabstop>
  <tabstop>orderTypeComboBox</tabstop>
  <tabstop>codeLineEdit</tabstop>
  <tabstop>hogaTypeComboBox</tabstop>
  <tabstop>qtySpinBox</tabstop>
  <tabstop>priceSpinBox</tabstop>
  <tabstop>orderBtn</tabstop>
  <tabstop>codeNameLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
